tests/pr_test.go

```diff
@@ -204,6 +204,54 @@ func TestRunUpgradeRoksPattern(t *testing.T) {
 	}
 }
 
+func TestValidateOverrideRoks(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsRoksPattern(t, "slzovr")
+	options.SkipTestTearDown = true
+	output, err := options.RunTestConsistency()
+	defer options.TestTearDown()
+
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+	assert.NotNil(t, options.LastTestTerraformOutputs, "Expected some Terraform outputs")
+
+	override_json := filepath.Join(roksPatternTerraformDir, "override.json")
```

Comment:
you won't want to test override if the RunTestConsistency had an error or the outputs is null. You could wrap this entire section in an `if{}` so that you only attempt it if "err" is nil and you have outputs.

Reply:
added